src/main/resources/view/MainWindow.fxml

```diff
@@ -11,9 +11,10 @@
 <?import javafx.scene.layout.VBox?>
 <?import javafx.stage.Screen?>
 <?import javafx.scene.layout.HBox?>
+<?import javafx.scene.control.TextField?>
 
 <fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1"
-         title="Address App" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
+         title="{{CLI}}Tutors" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
```

Comment:
I think we decided to get rid of the curly braces, so maybe let's just remove it here too